docs/reference/aggregations/bucket/histogram-aggregation.asciidoc

```diff
@@ -266,10 +267,10 @@ The above will sort the buckets based on the avg rating among the promoted produ
 ==== Offset
 
 By default the bucket keys start with 0 and then continue in even spaced steps of `interval`, e.g. if the interval is 10 the first buckets
-(assuming there is data inside them) will be [0 - 9], [10-19], [20-29]. The bucket boundaries can be shifted by using the `offset` option.
+(assuming there is data inside them) will be [0, 10), [10, 20), [20, 30). The bucket boundaries can be shifted by using the `offset` option.
 
-This can be best illustrated with an example. If there are 10 documents with values ranging from 5 to 14, using interval `10` will result in
-two buckets with 5 documents each. If an additional offset `5` is used, there will be only one single bucket [5-14] containing all the 10
+This can be best illustrated with an example. If there are 10 documents with integer values ranging from 5 to 14, using interval `10` will result in
+two buckets with 5 documents each. If an additional offset `5` is used, there will be only one single bucket [5, 15) containing all the 10
```

Comment:
For consistency, can you format this range as code (i.e. within `backticks`) please?